docs/public/replay-viewer/style.css

```
@@ -0,0 +1,565 @@
+
+.file-input {
+  position: relative;
+}
+
+.file-input[hidden] {
+  display: none !important;
+}
+
+.file-input input {
+  position: absolute;
+  inset: auto 0 0 0;
```

Stylelint (17.11.0)

[error] Expected "auto 0 0 0" to be "auto 0 0" (shorthand-property-no-redundant-values)

(shorthand-property-no-redundant-values)